crocket/src/main/java/com/crocket/CroquetController.java

```diff
@@ -0,0 +1,37 @@
+package com.crocket;
+
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+
+public class CroquetController implements KeyListener{
+    
+    private CroquetView view;
+    
+    private Ball ball;
+
+    public CroquetController(CroquetView view, Ball ball){
+        this.view = view;
+        this.ball = ball;
+        view.addKeyListener(this);
+    }
+    
+    public void keyPressed(KeyEvent ke){
+        if (ke.getKeyCode() == KeyEvent.VK_SPACE){
+            ball.startBall(4, 4, 5);
+            
+        }
+    }
+
+    public void keyReleased(KeyEvent ke){
```

Comment:
Is this method needed?

Reply:
yes, because controller needs to implement KeyListener to be able to add a key listener and that is a method from the interface